compiler/quilt/tools/store.py

```diff
@@ -315,15 +325,21 @@ def parse_package_extended(name):
                 else:
                     # usr/pkg:hashval
                     hash = versioninfo
-        owner, pkg, subpath = parse_package(name, allow_subpath=True)
+        team, owner, pkg, subpath = parse_package(name, allow_subpath=True)
     except ValueError:
         pkg_format = 'owner/package_name/path[:v:<version> or :t:tag or :h:hash]'
         raise CommandException("Specify package as %s." % pkg_format)
     return owner, pkg, subpath, hash, version, tag
 
 def parse_package(name, allow_subpath=False):
     try:
-        values = name.split('/')
+        values = name.split(':', 1)
+        if len(values) > 1:
+            team = values[0]
+        else:
+            team = None
+
+        values = values[-1].split('/')
```

Comment:
This splats on the special forms of package spec right? like user/pkg:h:hash

Reply:
need tests at the very least

Reply:
@dimaryaz thoughts?

Reply:
Yes, I forgot about that one. Not sure if we ever officially shipped that feature - but if we did, then yeah, need tests and a fix.

(Technically, though, it's the `parse_package_extended` function that needs to be fixed.)

Reply:
We've decided to backlog this - so ok for now